pkg/propertychecker/azure/checker_test.go

```diff
@@ -0,0 +1,295 @@
+            },
+            {
+                "family": "Standard",
+                "name": "Standard_B1s",
+                "size": "Standard_B1s"
+            }
+        ]
+    }
+}`
+
+func TestCheckIfMeetSKUCapacityRequirement(t *testing.T) {
+	// Prepare test data
+	cluster := &clusterv1beta1.MemberCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: map[string]string{
+				utils.AzureLocationLabelKey:       "centraluseuap",
+				utils.AzureSubscriptionIDLabelKey: "8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8",
+			},
+		},
+	}
+
+	tests := []struct {
+		name           string
+		cluster        *clusterv1beta1.MemberCluster
+		selector       placementv1beta1.PropertySelectorRequirement
+		mockStatusCode int
+		wantAvailable  bool
+		expectError    bool
+		errorSubstring string
+	}{
+		{
+			name:    "valid request",
+			cluster: cluster,
+			selector: placementv1beta1.PropertySelectorRequirement{
+				Name:     "kubernetes.azure.com/vm-sizes/Standard_D2s_v3/capacity",
+				Operator: placementv1beta1.PropertySelectorGreaterThan,
+				Values:   []string{"1"},
+			},
+			mockStatusCode: http.StatusOK,
+			wantAvailable:  true,
+			expectError:    false,
+		},
+		{
+			name:    "invalid capacity request",
+			cluster: cluster,
+			selector: placementv1beta1.PropertySelectorRequirement{
+				Name:     "kubernetes.azure.com/vm-sizes/Standard_D3s_v3/capacity",
+				Operator: placementv1beta1.PropertySelectorGreaterThan,
+				Values:   []string{"2"},
+			},
+			mockStatusCode: http.StatusOK,
+			wantAvailable:  false,
+			expectError:    false,
+		},
+		{
+			name: "missing Azure labels",
+			cluster: &clusterv1beta1.MemberCluster{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"other-label": "other-value",
+					},
+				},
+			},
+			selector: placementv1beta1.PropertySelectorRequirement{
+				Name:     "kubernetes.azure.com/vm-sizes/Standard_D2s_v3/capacity",
+				Operator: placementv1beta1.PropertySelectorGreaterThan,
+				Values:   []string{"10"},
+			},
+			expectError:    true,
+			errorSubstring: "failed to extract Azure location label from cluster : label \"fleet.azure.com/location\" not found in cluster",
+		},
+		{
+			name:    "unavailable SKU",
+			cluster: cluster,
+			selector: placementv1beta1.PropertySelectorRequirement{
+				Name:     "kubernetes.azure.com/vm-sizes/Standard_D2s_v4/capacity",
+				Operator: placementv1beta1.PropertySelectorGreaterThan,
+				Values:   []string{"10"},
+			},
+			mockStatusCode: http.StatusOK,
+			wantAvailable:  false,
+			expectError:    false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// Create mock server
+			server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				// Verify request method
+				if r.Method != http.MethodPost {
+					t.Errorf("got %s, want POST request", r.Method)
+				}
+
+				// Verify headers
+				if r.Header.Get("Content-Type") != "application/json" {
```

Comment:
wantong has already define the key as const in pkg/clients/httputil/httputil.go, please use them